public/static/locales/en/analyses.json

```
@@ -40,6 +40,7 @@
     "email": "Email",
     "emptyValue": "-",
     "endDate": "End date",
+    "extend": "Extend",
     "extendTime": "Extend time limit",
```

Not actually part of this PR, but since it was in the screenshot, perhaps:
```suggestion
    "extendTime": "Extend Time Limit",
```